src/components/icon/icon.jsx

```diff
@@ -19,9 +26,10 @@ export default function Icon(props) {
       id={props.id}
       onClick={props.onClick}
       role={props.role}
+      title={props.titleAttrText}
```

Comment:
SVGs don't use the title attribute. They use the nested `<title>` element or the `aria-label` attribute.

> SVG user agents MUST provide an accessible object in the accessibility tree for rendered SVG elements that meet any of the following criteria, unless they are excluded from the accessibility tree per the rules in Excluding Elements from the Accessibility Tree:
> - It has at least one direct child ‘title’ element or ‘desc' element that is not empty after trimming whitespace. User agents MAY include elements with these child elements without checking for valid text content.
> - It has a non-empty (after trimming whitespace) ‘aria-label’ attribute or ‘aria-roledescription’ attribute.
> - It has an ‘aria-labelledby’ attribute or ‘aria-describedby’ attribute containing valid IDREF tokens. User agents MAY include elements with these attributes without checking for validity.

From https://www.w3.org/TR/svg-aam-1.0/

You can test for this too by using user-centric queries (users only click on "visible text" or "screen reader text"):

```
screen.getByRole('img', { name: 'Accessible SVG Name' })
```

More information at https://github.com/testing-library/dom-testing-library/issues/685